plugins/by-name/ts-comments-nvim/default.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  ...
+}:
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "ts-comments-nvim";
+  originalName = "ts-comments.nvim";
+  luaName = "ts-comments";
+  package = "ts-comments-nvim";
+
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  settingsOptions = {
+    lang =
+      lib.nixvim.defaultNullOpts.mkAttrsOf lib.types.anything
+        {
+          astro = "";
+          axaml = "";
+          blueprint = "// %s";
+          c = "// %s";
+          c_sharp = "// %s";
+          clojure = [
+            ";; %s"
+            "; %s"
+          ];
+          cpp = "// %s";
+          cs_project = "";
+          cue = "// %s";
+          fsharp = "// %s";
+          fsharp_project = "";
+          gleam = "// %s";
+          glimmer = "{{! %s }}";
+          graphql = "# %s";
+          handlebars = "{{! %s }}";
+          hcl = "# %s";
+          html = "";
+          hyprlang = "# %s";
+          ini = "; %s";
+          ipynb = "# %s";
+          javascript = {
+            __default = "// %s";
+            "/* %s */" = null;
```

Comment:
Shouldn't this be unkeyed entries?

Reply:
Or is it an alternate syntax accepted by the plugin?

Reply:
Yeah, just late night stupidity